subworkflows/local/estimate_contamination.nf

```diff
@@ -48,6 +47,10 @@ workflow ESTIMATE_CONTAMINATION {
 
         ch_versions     = ch_versions.mix( BAM_DOCOUNTS_CONTAMINATION_ANGSD.out.versions )
         ch_angsd_contam = BAM_DOCOUNTS_CONTAMINATION_ANGSD.out.txt
+                            .map{
+                                meta, txt ->
+                                output: txt
+                            }
```

Comment:
The syntax you are using here is that of a `multiMap` and not a `map`. Please fix. 

I think this can be replaced with `.map { it[1] }` actually to drop the meta. Less explicit, but that's how we've done it elsewhere in the pipeline.